### ISSUE TITLE
About some details

### ISSUE THREAD

**Issue by Fzz123** [jxgu1016/MNIST_center_loss_pytorch#1]
In the code, loss is caculated by:
loss = criterion[0](pred, target) + loss_weight * criterion[1](target, ip1)

Why not
loss = criterion[0](pred, target) + loss_weight * criterion[1](target, ip1) / target.size(0)

**Comment by jxgu1016** [comment]
Please refer to the Eq. 5 in the paper.

**Comment by Fzz123** [comment]
Softmax loss in Pytorch (or other platform) use coverage operation, but the autor of this article didn't show it in the formula. So I think may be we should do this operation.
